wrappers/python/README.md

```diff
@@ -47,6 +47,16 @@ For development mode installation use
     pip install -e .
 ```
 
+You can also install the package from the dist folder with the examples
```

Comment:
Please rewrite the sentence so it is clear that the [examples] will install the extra requirements needed by the example code (in this case wnt examples)

Reply:
Done